src/main/java/domain/lotto/LottoService.java

```diff
@@ -0,0 +1,66 @@
+package domain.lotto;
+
+import domain.lottoTicket.LottoTicket;
+import domain.lottoTicket.LottoTicketService;
+import domain.lottoWinningStatistics.LottoWinningStatistics;
+import domain.lottoWinningStatistics.LottoWinningStatisticsService;
+import java.util.ArrayList;
+import java.util.List;
+import view.InputView;
+import view.OutputView;
+
+public class LottoService {
+    private final int LOTTO_PRICE = 1000;
+    private final InputView inputView;
+    private final  OutputView outputView;
+    public LottoService(InputView inputView, OutputView outputView) {
```

Comment:
```suggestion
private final  OutputView outputView;

public LottoService(InputView inputView, OutputView outputView) {
```
사소한 거지만 줄바꿈하면 가독성이 더 좋아질 거 같아요!